packages/website/src/bin/generate-pages.js

```diff
@@ -13,12 +13,14 @@ const packageRoot = path.resolve(__dirname, '..', '..');
 
```

Comment:
As an FYI, I'm currently copying over the logic from this file to a new place. After this gets merged I'll make sure this still works, but keep this and any other things you change in mine when reviewing my PRs :)